<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(244, 154, 186);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(244, 154, 186);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QPushButton" name="button_edit">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>425</y>
         <width>140</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: white;
    border: 1px solid lightgray;
    border-radius: 8px;
    padding: 6px 12px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #f4e1ec;
}
</string>
       </property>
       <property name="text">
        <string>Edit Profile</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_save">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>425</y>
         <width>140</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: white;
    border: 1px solid lightgray;
    border-radius: 8px;
    padding: 6px 12px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #f4e1ec;
}
</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>10</y>
         <width>301</width>
         <height>41</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	background-color: rgb(254, 227, 220);
	font: 20pt &quot;Source Code Pro&quot;;
	border: 2px solid indianred;
    border-radius: 10px;
    padding: 8px 10px;
}
  </string>
       </property>
       <property name="text">
        <string>Profile</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="labelavatar">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>60</y>
         <width>120</width>
         <height>120</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    background-color: #ffffff;
    border-radius: 60px;
    border: none;
    qproperty-alignment: AlignCenter;
}
</string>
       </property>
       <property name="text">
        <string>Avatar</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_1">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>195</y>
         <width>301</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
QLineEdit {
    background-color: white;
    border: 1px solid lightgray;
    border-radius: 10px;
    padding: 8px 10px;
    font-size: 14px;
}
</string>
       </property>
       <property name="inputMask">
        <string extracomment="Name"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>265</y>
         <width>301</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: white;
    border: 1px solid lightgray;
    border-radius: 10px;
    padding: 8px 10px;
    font-size: 14px;
}
</string>
       </property>
       <property name="inputMask">
        <string extracomment="Surname"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Surname</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>335</y>
         <width>301</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit {
    background-color: white;
    border: 1px solid lightgray;
    border-radius: 10px;
    padding: 8px 10px;
    font-size: 14px;
}
</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:14px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="placeholderText">
        <string>Short Bio...</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
